src/index.ts

```diff
@@ -25,5 +25,3 @@ export {
   MountingOptions,
   createWrapperError
 }
-
-export type { ComponentMountingOptions } from './mount'
```

Comment:
I think we should still export the type as it has been introduced with #2009. Should be type from here, right? https://github.com/vuejs/test-utils/blob/719549ddea62917629f7bc172a04694f8bde49d2/src/mount.ts#L38-L50

Reply:
Updated.